frontend/src/bundles/video-editor/components/timeline/subcomponents/timeaxis.tsx

```diff
@@ -0,0 +1,118 @@
+import { minutesToMilliseconds } from 'date-fns';
+import { useTimelineContext } from 'dnd-timeline';
+
+import { Box } from '../../../../common/components/components.js';
+import { useMemo } from '../../../../common/hooks/hooks.js';
+
+interface Marker {
```

Comment:
only types for props should be in the component file

also by default we use Type not Interface (A6 rule)- https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md#advance

Reply:
Thank you, changed interface for Type